package-release.sh

```diff
@@ -15,28 +15,27 @@ if [ -e "$NVOPTIX_BUILD_DIR" ]; then
   exit 1
 fi
 
-function build_arch {
-  export WINEARCH="win$1"
-  export WINEPREFIX="$NVOPTIX_BUILD_DIR/wine.$1"
-  
-  cd "$NVOPTIX_SRC_DIR"
-
-  meson --cross-file "$NVOPTIX_SRC_DIR/build-wine$1.txt"  \
-        --buildtype "release"                         \
-        --prefix "$NVOPTIX_BUILD_DIR/install.$1"         \
-        --libdir="lib$1"				\
-	--strip                                       \
-        "$NVOPTIX_BUILD_DIR/build.$1"
-
-  cd "$NVOPTIX_BUILD_DIR/build.$1"
-  ninja install
-
-  mv "$NVOPTIX_BUILD_DIR/install.$1/lib$1" "$NVOPTIX_BUILD_DIR"
-  mv "$NVOPTIX_BUILD_DIR/install.$1/fakedlls" "$NVOPTIX_BUILD_DIR/fakedlls$1"
-  cp "$NVOPTIX_SRC_DIR/setup_nvoptix.sh" "$NVOPTIX_BUILD_DIR/setup_nvoptix.sh"
-  chmod +x "$NVOPTIX_BUILD_DIR/setup_nvoptix.sh"
-  rm -R "$NVOPTIX_BUILD_DIR/build.$1"
-  rm -R "$NVOPTIX_BUILD_DIR/install.$1"
-}
-
-build_arch 64
+cd "$NVOPTIX_SRC_DIR"
+
+meson --cross-file "$NVOPTIX_SRC_DIR/build-wine64.txt"    \
+        --buildtype "release"                             \
+        --prefix "$NVOPTIX_BUILD_DIR/install"             \
+        --libdir="x64"                                    \
+	--strip                                           \
+        "$NVOPTIX_BUILD_DIR/build"
+
+cd "$NVOPTIX_BUILD_DIR/build"
+ninja install
+
+mv "$NVOPTIX_BUILD_DIR/install/x64" "$NVOPTIX_BUILD_DIR"
+
+if [ "$2" == "--fakedlls" ]; then
+  mv "$NVOPTIX_BUILD_DIR/install/fakedlls" "$NVOPTIX_BUILD_DIR/fakedlls"
```

Comment:
If the `.dll.so` has its own `x64` subdir, why does fakedll not have a one as well?

Or, from another point of view: what's the point of keeping a `x64` subdir for `.dll.so` if there is only one arch we can possibly support?

Reply:
What do you suggest? just `lib` instead of x64?

Reply:
Either just `lib` or no subdir at all. Unless we actually want to make it explicit that only `x86_64` is supported, in which case or we can borrow Wine's new convention:

```
x86_64-unix/nvoptix.dll.so
x86_64-windows/nvoptix.dll
```

(The second one being fakedll.)